docs/README.md

```diff
@@ -45,11 +45,18 @@ run the following command:
 brew install kube-linter
 ```
 
+### Using nix-shell
+To install by using [nix](https://nixos.org/) on macOS, Linux, and [Windows Subsystem for Linux (WSL)](https://docs.microsoft.com/en-us/windows/wsl/about),
+run the following command:
+```bash
+nix-shell -p kube-linter
+```
+
 ### Using Docker
 
 1. Get the latest KubeLinter Docker image:
    ```bash
-   docker pull stackrox/kube-linter:0.1.4
+   docker pull stackrox/kube-linter:v0.6.8
```

Comment:
Since this will be stale, just use latest here? We have the notion below that latest isn't recommended.

Reply:
```suggestion
   docker pull stackrox/kube-linter:latest
```